README.rst

```diff
@@ -29,6 +29,9 @@ Quick Links
 -----------
 
 * `Read the docs <https://falcon.readthedocs.io/en/stable>`_
+  (`FAQ <https://falcon.readthedocs.io/en/stable/user/faq.html>`_ -
+  `getting help <https://falcon.readthedocs.io/en/stable/community/help.html>`_ -
+  `reference <https://falcon.readthedocs.io/en/stable/api/index.html>`_)
```

Comment:
I like this better. not sure if `<...>` should be nicer to look at.

But it's fine even as is

Reply:
Heh, we could probably push the bikeshedding of parentheses to another PR :)